src/EprPrnIntegration.Api/UpdateProducersFunction.cs

```diff
@@ -65,7 +76,8 @@ private async Task HandleApiResponse(HttpResponseMessage pEprApiResponse, DateTi
     {
         if (pEprApiResponse.IsSuccessStatusCode)
         {
-            logger.LogInformation($"Producers list successfully updated in NPWD for time period {fromDate} to {toDate}.");
+            logger.LogInformation(
+                $"Producers list successfully updated in NPWD for time period {fromDate} to {toDate}.");
             return;
         }
```

Comment:
I think as part of this ticket similar changes need to be done on updateprns can we do that as well?

Reply:
done